src/frontend/src/index.ts

```diff
@@ -33,6 +34,25 @@ const readCanisterId = (): string => {
 };
 
 const init = async () => {
+  // Show version information for the curious programmer
+  try {
+    console.log("Welcome to Internet Identity!");
+    console.log(
+      "The code can be found here: https://github.com/dfinity/internet-identity"
+    );
+    console.log(
+      `https://github.com/dfinity/internet-identity/commit/${version.commit}`
+    );
+    if (version.release !== "") {
```

Comment:
here you check against `!== ""` but in `version.ts` it sets `undefined`. is that expected?

Reply:
last minute change, forgot to update here. Thanks a lot!

Reply:
fixed